src/components/mal/FremtidigSituasjon.tsx

```diff
@@ -0,0 +1,62 @@
+import * as React from 'react';
+import { useEffect, useState } from 'react';
+import './FremtidigSituasjon.less';
+import { hentTekst, SituasjonAlternativ, SporsmalType } from '../registreringsinfo/Alternativer';
+import { FremtidigSituasjonType } from '../../datatyper/fremtidigSituasjonType';
+import { Collapse } from 'react-collapse';
+import { AlternativGruppe } from './AlternativGruppe';
+import { hentFremtidigSituasjon, oppdaterFremtidigSituasjon } from '../../api/api';
+
+function FremtidigSituasjon () {
+    const [endreVisning, setSkalEndreState] = useState(false);
```

Comment:
Mulig dette er så mange state variabler at det er enklere å lage et state object, som man endrer i stedet for 1 og 1 variabel ?

Reply:
Enig

Reply:
Jeg har endret det nå men syns det var ryddigere sånn det var. Vi kan se på det sammen i morgen